client/evm-tracing/src/listeners/call_list.rs

```diff
@@ -519,6 +527,21 @@ impl Listener {
 				// behavior (like batch precompile does) thus we simply consider this a call.
 				self.call_type = Some(CallType::Call);
 			}
+			EvmEvent::Log {
+				address,
+				topics,
+				data,
+			} => {
+				let log = Log {
+					address,
+					topics,
+					data,
+				};
+				let key = self.entries_next_index.saturating_sub(1u32);
```

Comment:
`entries_next_index` is specific to `entries`. Maybe we can merge `entries` with `log_entries` into a single map.

Reply:
Mergine `entries` with `log_entries` ended up braking the log unit tests. Probably missing something in the updated version of the implementation [here](https://github.com/moonbeam-foundation/moonbeam/tree/aleks-debug-trace-call-options-logs-in-entries)